roles/satellite-clone/files/disassociate_capsules.rb

```diff
@@ -17,8 +17,8 @@
   reverse_commands = []
   external_capsules.each do |capsule|
     capsule[:lifecycle_environments].each do |env|
-      `hammer -u #{username} -p #{password} --csv capsule content remove-lifecycle-environment --id #{capsule[:id]} --environment #{env} --organization "#{capsule[:organization]}"`
-      reverse_command = "hammer -u #{username} -p #{password} --csv capsule content add-lifecycle-environment --id #{capsule[:id]} --environment #{env} --organization \"#{capsule[:organization]}\""
+      `hammer -u #{username} -p #{password} --csv capsule content remove-lifecycle-environment --id #{capsule[:id]} --environment "#{env}" --organization "#{capsule[:organization]}"`
+      reverse_command = "hammer -u #{username} -p #{password} --csv capsule content add-lifecycle-environment --id #{capsule[:id]} --environment \"#{env}\" --organization \"#{capsule[:organization]}\""
```

Comment:
why are the quotes escaped here and not in the line above?

Reply:
ah nvm, I see that is the pattern used with `capsule[:organization]` before, I'm not sure why we do this, I'll have to look more :man_shrugging:

Reply:
the one is executed directly, the other is stored to a var. in the second case, you already have `""` for the var assignment, so you can't use literal `"` inside the var

Reply:
thanks, I saw that after I made the comment :+1: